packages/components/psammead-navigation/hooks/useMediaQuery.js

```diff
@@ -0,0 +1,21 @@
+import { useState, useEffect } from 'react';
+
+const useMediaQuery = query => {
+  const isClient = typeof window !== 'undefined';
+
+  const mediaMatch = isClient ? window.matchMedia(query) : undefined;
+  const mediaMatches = isClient ? mediaMatch.matches : undefined;
+
+  const [matches, setMatches] = useState(mediaMatches);
+
+  useEffect(() => {
+    const handler = event => setMatches(event.matches);
+
+    mediaMatch.addListener(handler);
+    return () => mediaMatch.removeListener(handler);
+  });
```

Comment:
Is this going to re-run useEffect after every render? As you're just adding a listener when the component mounts, could you add `[]` as the second argument to useEffect?